configs/sites/derecho/packages.yaml

```diff
@@ -0,0 +1,163 @@
+#     - openmpi/4.1.1
+  python:
+    buildable: False
+    externals:
+    - spec: python@3.10.10+bz2+ctypes+dbm+lzma+nis+pyexpat+pythoncmd+readline+sqlite3+ssl+tix+tkinter+uuid+zlib
+      prefix: /glade/u/apps/derecho/23.06/opt
+  py-pip:
+    buildable: False
+    externals:
+    - spec: py-pip@22.0.4
+      prefix: /usr
+  cmake:
+    externals:
+    - spec: cmake@3.26.3
+      prefix: /glade/u/apps/derecho/23.06/spack/opt/spack/cmake/3.26.3/gcc/7.5.0/l2rq
+  coreutils:
+    externals:
+    - spec: coreutils@8.32
+      prefix: /usr
+  texinfo:
+    externals:
+    - spec: texinfo@7.0
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: texinfo@6.5
+      prefix: /usr
+  groff:
+    externals:
+    - spec: groff@1.22.4
+      prefix: /usr
+  automake:
+    externals:
+    - spec: automake@1.16.5
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: automake@1.15.1
+      prefix: /usr
+  gawk:
+    externals:
+    - spec: gawk@4.2.1
+      prefix: /usr
+  m4:
+    externals:
+    - spec: m4@1.4.18
+      prefix: /usr
+  subversion:
+    externals:
+    - spec: subversion@1.14.1
+      prefix: /usr
+  flex:
+    externals:
+    - spec: flex@2.6.4+lex
+      prefix: /glade/u/apps/derecho/23.06/opt
+  openssl:
+    externals:
+    - spec: openssl@1.1.1l
+      prefix: /usr
+  pkg-config:
+    externals:
+    - spec: pkg-config@0.29.2
+      prefix: /usr
+  git:
+    externals:
+    - spec: git@2.40.0~tcltk
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: git@2.35.3+tcltk
+      prefix: /usr
+  git-lfs:
+    externals:
+    - spec: git-lfs@3.3.0
+      prefix: /glade/u/apps/derecho/23.06/opt
+  meson:
+    externals:
+    - spec: meson@1.1.0
+      prefix: /glade/u/apps/derecho/23.06/opt
+  curl:
+    externals:
+    - spec: curl@8.0.1+nghttp2
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: curl@7.79.1+gssapi+ldap+nghttp2
+      prefix: /usr
+  libtool:
+    externals:
+    - spec: libtool@2.4.7
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: libtool@2.4.6
+      prefix: /usr
+  autoconf:
+    externals:
+    - spec: autoconf@2.71
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: autoconf@2.69
+      prefix: /usr
+  gmake:
+    externals:
+    - spec: gmake@4.2.1
+      prefix: /usr
+  bison:
+    externals:
+    - spec: bison@3.8.2
+      prefix: /glade/u/apps/derecho/23.06/opt
+    - spec: bison@3.0.4
+      prefix: /usr
+  binutils:
+    externals:
+    - spec: binutils@2.37.20211103
+      prefix: /usr
+  cvs:
+    externals:
+    - spec: cvs@1.12.13
+      prefix: /usr
+  findutils:
+    externals:
+    - spec: findutils@4.8.0
+      prefix: /usr
+  diffutils:
+    externals:
+    - spec: diffutils@3.6
+      prefix: /usr
+  openssh:
+    externals:
+    - spec: openssh@8.4p1
+      prefix: /usr
+  tar:
+    externals:
+    - spec: tar@1.34
+      prefix: /usr
+  boost:
+    buildable: False
+    externals:
+    - spec: boost@1.78.0
+      prefix: /glade/u/home/mpotts/boost
```

Comment:
We can't have user directories in the site configs. Why do we need a separate boost and doxygen?

Reply:
And, wouldn't you know it. Just after I typed that, an email came in with access to the role account!

Reply:
Let's see if we can fix the boost issues somehow. doxygen might be available from the OS?